services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/docker_compose_utils.py

```diff
@@ -46,7 +54,7 @@ async def _write_file_and_spawn_process(
 
 
 async def docker_compose_config(
-    compose_spec_yaml: str, settings: ApplicationSettings, timeout: Optional[int] = None
+    compose_spec_yaml: str, *, timeout: int = 60
```

Comment:
is this 60s timeout there for a reason? why remove the None option? the client anyway has a timeout too right?

Reply:
The None is an artefact of refactoring.
The 60 seconds is there since sometimes the validation can hang, especially if you send a spec which is not valid. Usually it's a very fast operation.